### PR TITLE
Modifying addblast

### PR DESCRIPTION
**Pull request by HsiuKangHuang**
modified the argument of manage.py addblast for adding description of blastdb.

### CODE REVIEW

**Comment by mpoelchau** [comment]
Thanks @HsiuKangHuang!

Can you:
 - fix the Codacy issue (trailing whitespace: https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)
- We'll have to figure out the travis problem together.
- Currently the description, on the user interface side, displays with brackets and apostrophes - for example, if my description is TEST TEST, it will show up as ['TEST TEST']. Is there a way to get rid of the brackets and apostrophes?

File: blast/management/commands/addblast.py [hunk 1]
```diff
@@ -21,7 +22,13 @@ def handle(self,*args,**options):
...
             title = options['filename'][0]
             fasta_file_path = get_path('blast',title)
-            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = '', is_shown = False )
+
+            if options['description']=='':  
+               options['description']= options['filename'][0] 
+            else:
+               options['description']= ' '.join(options['description'])
+                              
+            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = options['description'], is_shown = False )
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [line too long (170 > 159 characters)](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           

Complexity increasing per file
==============================
- blast/management/commands/addblast.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088&bid=15051910)

File: blast/management/commands/addblast.py [hunk 2]
```diff
@@ -21,7 +22,13 @@ def handle(self,*args,**options):
             blast_type = get_type(dataset,molecule2,molecule_str,dataset_str)
             title = options['filename'][0]
             fasta_file_path = get_path('blast',title)
-            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = '', is_shown = False )
+
+            if options['description']=='':
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

File: blast/management/commands/addblast.py (hunk 1 above)
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

File: blast/management/commands/addblast.py [hunk 3]
```diff
@@ -21,7 +22,13 @@ def handle(self,*args,**options):
             blast_type = get_type(dataset,molecule2,molecule_str,dataset_str)
             title = options['filename'][0]
             fasta_file_path = get_path('blast',title)
-            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = '', is_shown = False )
+
+            if options['description']=='':  
+               options['description']= options['filename'][0]
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

File: blast/management/commands/addblast.py [hunk 4]
```diff
@@ -21,7 +22,13 @@ def handle(self,*args,**options):
             blast_type = get_type(dataset,molecule2,molecule_str,dataset_str)
             title = options['filename'][0]
             fasta_file_path = get_path('blast',title)
-            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = '', is_shown = False )
+
+            if options['description']=='':  
+               options['description']= options['filename'][0] 
+            else:
+               options['description']= ' '.join(options['description'])
+
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

File: blast/management/commands/addblast.py [hunk 5]
```diff
@@ -10,6 +10,7 @@ def add_arguments(self,parser):
         parser.add_argument('Genus_Species',nargs='+',type=str)
         parser.add_argument('-t','--type',nargs='+',type=str,help='please enter nucleotide or peptide and enter Genome Assembly or Protein or Transcript')
         parser.add_argument('-f','--filename',nargs=1,type=str)
+        parser.add_argument('-d','--description',nargs='*',type=str,default='',help='please enter description')
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- blast/management/commands/addblast.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088&bid=15051910)

File: blast/management/commands/addblast.py [hunk 6]
```diff
@@ -21,7 +22,11 @@ def handle(self,*args,**options):
             blast_type = get_type(dataset,molecule2,molecule_str,dataset_str)
             title = options['filename'][0]
             fasta_file_path = get_path('blast',title)
-            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = '', is_shown = False )
+            if options['description']=='':
+               options['description']= options['filename'][0]
+            else:
+               options['description']= ' '.join(options['description'])
+            new_db = BlastDb(organism = organism, type = blast_type, fasta_file = fasta_file_path, title = title, description = options['description'], is_shown = False)
```
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [line too long (170 > 159 characters)](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

File: blast/management/commands/addblast.py (hunk 5 above)
Comment (hsiaoyi0504) [inline blast/management/commands/addblast.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088)

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- blast/management/commands/addblast.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4453088&bid=15051910)

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- blast/management/commands/addblast.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/genomics-workspace/genomics-workspace/pullRequest?prid=4453088&bid=15051910)

**Comment by mpoelchau** [comment]
This is great so far @HsiuKangHuang. I added some minor changes to this branch. Can you add one more section to the documentation, on the 'delete' CLI function? We currently don't use this in our workflows but it would be good to document in the genomics-workspace documentation. In the below examples, `-o` means to delete the organism, `-b` the blast database, and `-h` the hmmer database. Perhaps put this in a new section at the bottom of the documentation?


python manage.py delete -o Aethina tumida

python manage.py delete -b GCF_001937115.1_Atum_1.0_genomic.fna

python manage.py delete -h GCF_001937115.1_Atum_1.0_genomic.fna

**Pull request merged**